### PR TITLE
feature: error state select input

### PR DESCRIPTION
**Pull request by mhomolak**
- Error select for Console
- Addressed the spacing issue Nathan brought up by making all error-labels display: block
- Changed disabled fancy placeholder font-size to 14px per docs
- Small refactors; removed unnecessary @include from basic select (overzealous copy pasting on my part most likely) and moved pe-textLabelInput__label { display: etc } to where it's defined the first time instead of repeating multiple.

### CODE REVIEW

File: scss/elements/inputs/_basic-input-mixins.scss [hunk 1]
```diff
@@ -65,35 +65,51 @@
 
 };
 
+.container-extend {
```
Comment (JasonLantz) [inline scss/elements/inputs/_basic-input-mixins.scss]: i assume these classes are abstract... if that's true then the % operator could be used for creating an abstract class... there's an example in buttons... but this is fine.

Comment (mhomolak) [inline scss/elements/inputs/_basic-input-mixins.scss]: ok cool I'll fix that

File: scss/elements/buttons/_buttons.scss [hunk 2]
```diff
@@ -47,7 +47,7 @@
 
 // for fake links
 
-.pe-link-extend {
+%pe-link-extend {
```
Comment (wyseguyonline) [inline scss/elements/buttons/_buttons.scss]: I like this.  Granted the % should be a dead give-away that it is intended to be used with @extend, the use of "extend" in the name makes it more obvious.  I wonder if this should be added to our code standards doc?

Comment (JasonLantz) [inline scss/elements/buttons/_buttons.scss]: yes... polymorphism should be used if possible...

**Comment by udhadpa** [comment]
Tests written and verified. Good to merge

**Pull request merged**